src/app/settings/admin/presets/dialog/steps/preset/template.html

```diff
@@ -30,18 +30,33 @@
   </mat-form-field>
 
   <div class="optional-section">Optional Settings</div>
-
-  <mat-form-field fxFlex>
-    <mat-label>Restrict Preset Group to Domain</mat-label>
-    <input matInput
-           [formControlName]="controls.Domain"
-           [pattern]="domainRegex"
-           type="text"
-           title="domain"
-           autocomplete="off">
-    <mat-error *ngIf="form.get(controls.Domain).hasError( 'pattern')">
+  <mat-form-field class="domains-chip-list"
```

Comment:
I would reuse a `km-tag-list` and adjust it if needed to accept more parameters. This is basically a copy of the code that is already there. We can have a single generic shared component.

Reply:
Maybe that is something that we can do in a follow-up, separate PR? Because we are currently using `mat-chip-list` at other places as well. That can be refactored in one go to use a more generic shared component.

Reply:
Btw that stuff needs to be refactored anyways since right now if you try to create a data-center, it doesn't perform any validations for emails.

Reply:
Updated